app/controllers/workflow_controller.rb

```diff
@@ -0,0 +1,37 @@
+class WorkflowController < ApplicationController
+  before_action :check_privileges
+  before_action :get_session_data
+
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  include Mixins::GenericListMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::GenericShowMixin
+  include Mixins::BreadcrumbsMixin
+
+  def self.model
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook
+  end
+
+  def show_list
+    assert_privileges('embedded_workflow_view')
```

Comment:
@Fryguy we'll want this to be `embedded_configuration_script_payload_view` right?